src/js/components/DataTable/stories/ColumnSize.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, DataTable, Heading } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+export const DATA = [
+  {
+    name: 'Alan Josiah Werner Shirleen Foy',
+    location: 'Winston Salem',
+    date: '2018-01-09',
+    percent: 24,
+    paid: 3425,
+  },
+  {
+    name: 'Bryan Lane Smallwood Dion Gunderson',
+    location: 'Fort Collins',
+    date: '2018-06-10',
+    percent: 30,
+    paid: 1234,
+  },
+  {
+    name: 'Chris Willa Koehler Rocco Bales',
+    location: 'Palo Alto',
+    date: '2018-06-09',
+    percent: 40,
+    paid: 2345,
+  },
+  {
+    name: 'Eric Maegan Regalado Kiana Lawton',
+    location: 'Palo Alto',
+    date: '2018-06-11',
+    percent: 80,
+    paid: 3456,
+  },
+  {
+    name: 'Doug Yong Cleveland Jule Gantt',
+    location: 'Fort Collins',
+    date: '2018-06-10',
+    percent: 60,
+    paid: 1234,
+  },
+  {
+    name: 'Jet Isabella Mcnutt Deedee Bernstein',
+    location: 'Palo Alto',
+    date: '2018-06-09',
+    percent: 40,
+    paid: 3456,
+  },
+  {
+    name: 'Michael Corazon Ragan September Hynes',
+    location: 'Boise',
+    date: '2018-06-11',
+    percent: 50,
+    paid: 1234,
+  },
+  {
+    name: 'Tracy Kimbery Mccrary Jona Kinsey',
+    location: 'San Francisco',
+    date: '2018-06-10',
+    percent: 10,
+    paid: 2345,
+  },
+];
+
+const columnsThemeSize = [
```

Comment:
Not sure what the user should understand from the name that includes theme while there is no theme involved in this example. Could you please explain the thought behind it?

Reply:
I meant the sizes "small", "medium", "large" etc the come from the theme definition.

Please suggest if there's a better name for these size definitions - like say "standard sizes"?

Reply:
"standard sizes" sounds like a better fit for me. thanks!